app/views/components/_highlight-boxes.html.erb

```diff
@@ -4,16 +4,17 @@
   inverse_class = "app-c-highlight-boxes--inverse" if inverse
 %>
 <% if items.any? %>
-  <ol class="app-c-highlight-boxes">
-    <% items.each do |content_item| %>
+  <ol class="app-c-highlight-boxes" <%= "data-module=track-click" if local_assigns[:data_track_category] %>>
+    <% items.each_with_index do |content_item, index| %>
       <li class="app-c-highlight-boxes__item-wrapper">
         <div class="app-c-highlight-boxes__item <%= inverse_class %>">
-          <a
-            class="app-c-highlight-boxes__title <%= "app-c-highlight-boxes__title--featured" if content_item[:link][:featured] %>"
-            href="<%= content_item[:link].fetch(:path) %>"
-          >
-            <%= content_item[:link].fetch(:text)  %>
-          </a>
+          <%= link_to(
+            content_item[:link].fetch(:text),
+            content_item[:link].fetch(:path),
+            class: "app-c-highlight-boxes__title #{"app-c-highlight-boxes__title--featured" if content_item[:link][:featured]}",
+            data: (highlight_box_tracking_attributes(local_assigns[:data_track_category], content_item, index))
```

Comment:
This function feels like something that should be called in the presenter rather than the view.